diff --git gcc/config/riscv/linux.h gcc/config/riscv/linux.h
index 4afef7c22..e34746919 100644
--- a/gcc/config/riscv/linux.h
+++ b/gcc/config/riscv/linux.h
@@ -62,8 +62,4 @@ along with GCC; see the file COPYING3.  If not see
     %{static:-static} %{static-pie:-static -pie --no-dynamic-linker -z text}}"
 
-#define STARTFILE_PREFIX_SPEC 			\
-   "/lib" XLEN_SPEC "/" ABI_SPEC "/ "		\
-   "/usr/lib" XLEN_SPEC "/" ABI_SPEC "/ "	\
-   "/lib/ "					\
-   "/usr/lib/ "
+#define STARTFILE_PREFIX_SPEC ""

